packages/curvenote/src/sync/init.ts

```diff
@@ -115,48 +118,66 @@ export async function init(session: ISession, opts: Options) {
   let pullComplete = false;
   let title = projectConfig?.title || siteConfig.title || undefined;
   if (content === 'folder') {
-    if (projectConfigPaths.length) {
+    if (projectConfigPaths.length > 0) {
       const pathListString = projectConfigPaths
         .map((p) => `  - ${join(p, CURVENOTE_YML)}`)
         .join('\n');
+
       session.log.info(
         `👀 ${chalk.bold(
           'Found existing project config files on your path:',
         )}\n${pathListString}\n`,
       );
     }
-    if (!opts.yes) {
-      const promptTitle = await inquirer.prompt([questions.title({ title: title || '' })]);
-      title = promptTitle.title;
-    }
-    if (!projectConfig) {
-      try {
-        loadProjectFromDisk(session, path);
-        session.log.info(`📓 Creating project config`);
-        projectConfig = getDefaultProjectConfig(title);
-        projectConfigPaths.unshift(path);
-      } catch {
-        if (!projectConfigPaths.length) {
-          throw Error(`No markdown or notebook files found`);
+    if (opts.writeToc && projectConfigPaths.length > 0) {
+      if (validateTOC(session, currentPath)) {
+        session.log.warn('Not writing the table of contents, it already exists!');
+        return;
+      } else {
+        const project = projectFromPath(session, currentPath);
+        session.log.info(
+          `📓 Writing '_toc.yml' file to ${
+            currentPath === '.' ? 'the current directory' : currentPath
+          }`,
+        );
+        if (!project) throw Error(`Could not load project from ${currentPath}`);
+        writeTocFromProject(project, currentPath);
```

Comment:
Ok - confirmed that this does happen. If `site` config exists looking something like this:
```yaml
...
site:
  projects:
    - path: subproject
      slug: sub
```
It will still just try to write the `_toc` in the current folder, as opposed to `subproject` folder. That's a bit of an edge case though, I think it's fine to leave as is.

Reply:
ok - out of curiosity though, the change here would be to write toc where? to `projectConfigPaths[0]`? or individually write a toc to each [sub]path?

Reply:
happy to take this PR as is though as it meets the initial local use case we're trying to cover just now